### PR TITLE
feat: updated eula terms after onboarding

### PR DESCRIPTION
**Pull request by wadeking98**
Added ability to display updated terms on app startup after onboarding is completed
Depends on: https://github.com/openwallet-foundation/bifold-wallet/pull/1114

### CODE REVIEW

File: app/src/screens/Terms.tsx [hunk 1]
```diff
@@ -23,12 +24,15 @@ const digitalTrustHomeUrl = 'https://digital.gov.bc.ca/digital-trust/'
 const bcWebPrivacyUrl = 'https://www2.gov.bc.ca/gov/content/home/privacy'
 const digitalWalletPrivacyUrl = 'https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet/privacy'
 
+export const TermsVersion = '1.0.1'
```
Comment (cvarjao) [inline app/src/screens/Terms.tsx]: please update to `2.0`

File: app/src/screens/Terms.tsx [hunk 2]
```diff
@@ -96,85 +100,107 @@ const Terms = () => {
   return (
     <SafeAreaView edges={['left', 'right', 'bottom']}>
       <ScrollView style={[style.container]}>
-        <InfoTextBox>Please agree to the terms of use below before using this application.</InfoTextBox>
+        <InfoBox
+          notificationType={InfoBoxType.Info}
+          description={
+            agreedToPreviousTerms &&
+            'BC Wallet has updated the end user licencse agreement. To continue using BC Wallet, please reveiw and accept the updated terms.'
```
Comment (cvarjao) [inline app/src/screens/Terms.tsx]: Typo "licencse", and "reveiw"

File: app/src/screens/Terms.tsx (hunk 1 above)
Comment (wadeking98) [reply app/src/screens/Terms.tsx]: got it, I'll do that once the bifold pr is merged bc I'll have to update the package.json anyway

File: app/src/screens/Terms.tsx [hunk 3]
```diff
@@ -96,85 +100,108 @@ const Terms = () => {
   return (
     <SafeAreaView edges={['left', 'right', 'bottom']}>
       <ScrollView style={[style.container]}>
-        <InfoTextBox>Please agree to the terms of use below before using this application.</InfoTextBox>
+        <InfoBox
+          notificationType={InfoBoxType.Info}
+          description={
+            agreedToPreviousTerms
+              ? 'BC Wallet has updated the end user licencse agreement. To continue using BC Wallet, please reveiw and accept the updated terms.'
```
Comment (cvarjao) [inline app/src/screens/Terms.tsx:107]: Typos: "licencse", "reveiw"

File: app/src/screens/Terms.tsx [hunk 4]
```diff
@@ -23,12 +24,15 @@ const digitalTrustHomeUrl = 'https://digital.gov.bc.ca/digital-trust/'
 const bcWebPrivacyUrl = 'https://www2.gov.bc.ca/gov/content/home/privacy'
 const digitalWalletPrivacyUrl = 'https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet/privacy'
 
+export const TermsVersion = '2'
```
Comment (cvarjao) [inline app/src/screens/Terms.tsx:27]: I think `2.0` is a good balance, in case we do have minor updates.

**Review by bryce-mcmath — APPROVED** [review]
LGTM

**Pull request merged**